portal/src/main/webapp/js/src/oncoprint/webgl/dist/oncoprint-bundle.js

```diff
@@ -1656,14 +1657,17 @@ var OncoprintLabelView = (function () {
 		} else {
 		    var hovered_track = isMouseOnLabel(view, evt.pageY - view.$canvas.offset().top);
 		    if (hovered_track !== null) {
-			var $tooltip_div = $('<div>');
+			var $tooltip_div = $('<section>');
```

Comment:
It makes no difference to browsers yet. What do you think, should I use an `<aside>`, or leave out the semantic annotations altogether for now, reverting it to the inline element `<b>`?